src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -110,8 +112,17 @@ public override Type DefaultType
                 {
                     if (Context.Access == FileAccess.Read)
                     {
-                        return string.Compare("binary", Context.DataType, StringComparison.OrdinalIgnoreCase) == 0
+                        Type type = string.Compare("binary", Context.DataType, StringComparison.OrdinalIgnoreCase) == 0
                             ? typeof(byte[]) : typeof(string);
+
+                        if (string.Compare("many", Context.Cardinality, StringComparison.OrdinalIgnoreCase) == 0)
+                        {
+                            // arrays are supported for both trigger input as well
+                            // as output bindings
+                            type = type.MakeArrayType();
```

Comment:
This should just work for byte[][] bindings as well, once we pull in Mike's recent changes for supporting EventHub byte[][] bindings. In Functions you'd specify `dataType: "binary"` and `cardinality: "many"` to achieve that.